### PR TITLE
Add fixedWidth attribute to GridLayout

### PR DESCRIPTION
**Pull request by mattiasbpersson**
Suggested solution to: https://github.com/jbaysolutions/vue-grid-layout/issues/184

### CODE REVIEW

**Comment by mattiasbpersson** [comment]
If you have a better name for the attribute or the documentation, please get back and I'll update it asap.

**Comment by gmsa** [comment]
Sorry for the late reply, but I can't see how this is working. Its not making the GridItems square.

File: src/GridItem.vue [hunk 1]
```diff
@@ -664,7 +675,11 @@
             },
             // Helper for generating column width
             calcColWidth() {
-                var colWidth = (this.containerWidth - (this.margin[0] * (this.cols + 1))) / this.cols;
+                if (this.fixedWidth) {
+                    var colWidth = this.rowHeight;
```
Comment (mattiasbpersson) [inline src/GridItem.vue]: @gmsa this is where the colWidth is set to the same value as rowHeight. Which in turn makes it possible to create GridItems square. Then it's up to the user set same width and height of each GridItem.

Comment (mattiasbpersson) [reply src/GridItem.vue]: We had other solutions to this too, like setting the actual rowHeight the same way as the colWidth. If that is easier to understand we can update the PR. From my Point of view the important thing is have control over the colWidth too (doesn't have to be square in my case).

**Comment by gmsa** [comment]
This is what happens when I enable fixedWidth. If this is the intended behavior, I can't really see its purpose.

fixed-width

**Comment by mattiasbpersson** [comment]
Yes! You are able to have dependency between width and height. You couldn't before. It looks a bit strange in the test environment since the griditems are so small. If they are more 100px maybe it makes more sense?

I'll try to explain my use-case. 

We indend to be build a radiator showing statuses of jobs running in our ci-environement. For those jobs we need to display different amount of information. We don't want information to show differently depending on the screen resolution (except if resolution is very low). In these cases we would need to specify a fixed size of griditem to  be able to control this. We can then use the x/y value to specify the exact size we need.

**Pull request closed without merging**